### ISSUE TITLE
rxjs-no-unsafe-scope template string with default exported functions

### ISSUE THREAD

**Issue by kctang** [cartant/rxjs-tslint-rules#70]
Getting false positive with this scenario.

This is slightly more complicated to simulate as it requires a file with default exported function. This may not be a typical use case but that is how [graphql-tag](https://github.com/apollographql/graphql-tag) is used.

```typescript
// --- myFunc.ts
import { of } from 'rxjs'

export default function (s: any) {
  return of(null)
}

// --- usage in some other .ts file
  import myFunc from './myFunc'

  const defaultExportTemplateLiterals_notOk = of(null).pipe(
    // Unsafe scopes are forbidden
    concatMap(() => myFunc`bob`)
  )

// (workarounds removed as it does not really work)
```

Looking at how fast you resolve issues reported definitely motivates me to spend more time to submit issues that is easier to reproduce/simulate. Thanks!

**Comment by cartant** [comment]
Should be fixed in 4.14.3.
